src/stockwatch/use_cases/importing.py

```diff
@@ -122,10 +138,29 @@ def _process_valuta_exchange_row(row: dict[str, str]) -> CurrencyExchange:
     return CurrencyExchange(exchange_datetime, rate, amount_from)
 
 
+def _process_cash_settlement_row(
+    row: dict[str, str],
+    exchanges: list[CurrencyExchange],
+) -> CashSettlement:
+    date_time_str = row["Datum"] + ";" + row["Tijd"]
+    settlement_datetime = datetime.strptime(date_time_str, "%d-%m-%Y;%H:%M")
+    isin = IsinStr(row["ISIN"])
+    curr = row["Mutatie"]
+    amount = Amount(float(row["Bedrag"].replace(",", ".")), curr)
+    if curr != "EUR":
+        amount = apply_exchange(
+            settlement_datetime,
+            amount,
+            exchanges,
+        )
+    return CashSettlement(settlement_datetime, isin, amount)
```

Comment:
I'm not exactly sure why we need this CashSettlement class. One use-case I can think of, is that a delisting will cause a different sell amount as the current value of the stock.

If this is the case we're trying to catch I think we have a bit too much magic going on with the exchange rates. Also it is a bit unexpected that the `price` attribute of the `ShareTransaction` doesn't change.

Reply:
I try to not do magic but logic:

- we have a delisting-sell row, which unfortunately has no amount sold (not due to exchanges, but because it is specified as 0)
- the amount sold can be found in a cash settlement row
- the cash settlement row can be specified in non-EUR, in which case we will apply the corresponding exchange before 

Does this help? Shall I put that in a comment?

- for positions where the number of shares has become zero, the price has no meaning anymore (we do not know it either), only the realized result is interesting

Reply:
okay I think i'm starting to understand. I got confused mainly due to the `apply_exchanges` that was not necessary, the way it's done now makes sense I think.